src/coreclr/nativeaot/Runtime/FinalizerHelpers.cpp

```diff
@@ -58,6 +55,11 @@ uint32_t WINAPI FinalizerStart(void* pContext)
     UInt32_BOOL fResult = PalSetEvent(hFinalizerEvent);
     ASSERT(fResult);
 
+    if (YieldProcessorNormalization::IsMeasurementScheduled())
```

Comment:
This will be called only once at finalizer thread set up. Once we call into managed finalizer loop, we will not be here again.  
If the idea is to re-check this once in a while, a similar check should also be in `RhpSignalFinalizationComplete`. That is called when finalizer thread has drained the queue and has nothing else to do.

Reply:
The initial check should be before we wait on `hFinalizerEvent`, otherwise we will not do the first check until first finalizer runs, which may be never.

Reply:
Thanks, done